projects/angular/src/forms/datepicker/date-container.ts

```diff
@@ -96,10 +110,15 @@ import { ViewManagerService } from './providers/view-manager.service';
 export class ClrDateContainer extends ClrAbstractContainer implements AfterViewInit {
   focus = false;
 
+  @ContentChild(ClrDateInput) clrDateEl: ClrDateInput;
+  @ContentChild(ClrDateRangeStartInput) clrRangeStartEl: ClrDateRangeStartInput;
+  @ContentChild(ClrDateRangeEndInput) clrRangeEndEl: ClrDateRangeEndInput;
```

Comment:
```suggestion
  @ContentChild(ClrDateInput) clrDateInput: ClrDateInput;
  @ContentChild(ClrDateRangeStartInput) clrStartDateInput: ClrDateRangeStartInput;
  @ContentChild(ClrDateRangeEndInput) clrEndDateInput: ClrDateRangeEndInput;
```

The `El` suffix here is incorrect because the value is the directive instance, not an element.